test/python/project.py

```diff
@@ -51,13 +51,13 @@ def is_removed_on_pack(filename):
 # Class Project
 ###############
 class Project:
-    def __init__(self, name, basepath, template='cartridge'):
+    def __init__(self, cartridge_cmd, name, basepath, template='cartridge'):
```

Comment:
Could we eliminate passing this fixture in each test if you've already used it in "Project" class? Seems we could export it as Project.cmd or smth similar.

Reply:
I'm not sure that `cartridge_cmd` is a part of Project.

Passing this fixture in each tests is clear and doesn't confuse.

Reply:
IMO If you have a "Project" you should have a way to run it

Reply:
Yes, but `cartridge_cmd` doesn't belong to a project.

Reply:
But you've already used it to build. I don't see reasons why it should stop us from using to run.

Reply:
I sure that `project.cartridge_cmd` will confuse.
And I don't see any problems in passing this fixture, this command is the main object of testing, why shouldn't we pass it explicitly?

Reply:
ok